class UserNotFoundException(Exception):
    pass


class ActionNotFoundException(Exception):
    pass
